stories/core/designGrid.css

```diff
@@ -55,7 +56,7 @@
     background-image: linear-gradient(
       90deg,
       transparent var(--col),
-      color(var(--blueBerry) alpha(0.1)) var(--gutter)
+      color(var(--bolt) alpha(0.1)) var(--gutter)
```

Comment:
naming convention here should be theme based (ie, `primary1`) to avoid needing to update all references to colors any time the theme colors change.